mitmproxy-addons/ipcEventsAddon.py

```diff
@@ -0,0 +1,55 @@
+"""
+This is an addon for mitmproxy that sends events as nodejs-compatible IPC messages to a pipe.
+
+The file descriptor of the pipe to send the messages to needs to be configured as a mitmproxy option: ipcPipeFd
+If the option is not set, the addon will do nothing.
+
+To use the addon, start mitmproxy like this:
+
+mitmproxy -s ipc_events_addon.py --set ipcPipeFd=42
```

Comment:
That's the wrong file name.

Also, I'm not so sure how I feel about camel-cased files in dash-cased folders, but anyway. :D